### PR TITLE
Add extra2 content, data analysis, in Japanese and English.

### PR DESCRIPTION
**Pull request by wakky927**
## What
SSIA

## CHECKS :warning:

Please make sure you are aware of the following.

- [x] **The changes in this PR doesn't have private information

### CODE REVIEW

File: document/extra-02-data-analysis.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Extra2 では extra ディレクトリ配下の [extra-02-data-analysis-ja.ipynb](../extra/extra-02-data-analysis-ja.ipynb) にてデータ分析を行います。Notebook にある問題に取り組んで下さい。
+なお作業環境は自由ですが、ここではデータ分析でよく使用される `Google Colaboratory` について共有しておきます。
+
+## Google Colaboratory
+
+**:book: Reference**
+
+* (EN) [Welcome To Colaboratory](https://g.co/kgs/Z6J7Mct)
+* (JA) [Colaboratory へようこそ](https://g.co/kgs/3R2GzaR)
```
Comment (cocodrips) [inline document/extra-02-data-analysis.md:14]: Please add these contents
```suggestion
* (EN) [Welcome To Colaboratory](https://g.co/kgs/Z6J7Mct)
* (JA) [Colaboratory へようこそ](https://g.co/kgs/3R2GzaR)
* (EN) [Python for Data Science and Machine Learning Bootcamp](https://mercari.udemy.com/course/python-for-data-science-and-machine-learning-bootcamp/)
* (JA) [実践 Python データサイエンス](https://mercari.udemy.com/course/python-jp/)
* (JA) [データ分析コンペで楽しみながら学べるPython×データ分析講座](https://mercari.udemy.com/course/python-data/)
```

File: extra/extra-02-data-analysis-en.ipynb [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Problem 2-3\n",
+    "\n",
+    "Divide the `wine_df` into groups by `target` i.e. wine type and calculate the average value of `flavanoids`. (*Hints are [here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.groupby.html))\n",
+    "\n",
+    "またその結果、平均値が一番小さい `target` はどれですか？"
```
Comment (cocodrips) [inline extra/extra-02-data-analysis-en.ipynb:424]:
```suggestion
  "What is the target with the smallest average as a result?"
```

**Review by cocodrips — COMMENTED** [review]
I made a small comment, could you please correct that part? 

Also, since I'm thinking of distributing the answers via Drive, could you please temporarily remove them from GitHub?

File: extra/extra-02-data-analysis-en.ipynb (hunk 2 above)
Comment (wakky927) [reply extra/extra-02-data-analysis-en.ipynb:424]: コピーして英訳したのバレましたねww（見つけていただきありがとうございますmm）

**Review by cocodrips — APPROVED** [review]

**Review by cocodrips — APPROVED** [review]
Could you add the document link here?
https://github.com/mercari-build/mercari-build-training/blob/main/README.md?plain=1#L40

except it, LGTM!

**Pull request merged**